xml/System.Numerics/ILogarithmicFunctions`1.xml

```diff
@@ -119,9 +119,11 @@
         <Parameter Name="x" Type="TSelf" />
       </Parameters>
       <Docs>
-        <param name="x">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="x">The value to which one is added before computing the base-10 logarithm.</param>
+        <summary>Computes the base-10 logarithm of a value plus one.</summary>
+        <returns>
+          <c>log<sub>10</sub>(<paramref name="x" /> + 1)</c>
```

Comment:
These actually [look good in the docs](https://review.docs.microsoft.com/en-us/dotnet/api/system.numerics.ilogarithmicfunctions-1.log2p1?view=net-7.0&branch=pr-en-us-8185) - but are you okay with having HTML in the XML comments? How does it look in IntelliSense?